### DISCUSSION TITLE
Glossary?

### DISCUSSION THREAD

**Discussion by mwapst** [ivoa/ProposalDM#5] [General]
I think there’s a good case for a glossary, the DMs don’t really give enough business information, for example

- given that I can see that an Observation consists of a Target with a TechnicalGoal, what is the business meaning of these relationships {ObservingProposal --- Target},  {ObservingProposal --- TechnicalGoal}, 

- what is the business relationship between an Observatory/ Instrument/ SpectralWindowSetup and an Observation/ TechnicalGoal/ ScienceSpectralWindow? In which circumstances is it zero?

- how does TechnicalGoal relate to observatory equipment, ie given I define the performance parameters how do I know which instrument is capable of meeting them, and which observatories have such equipment - this would drive my submission choice

- what is the business meaning of Field extensions such as Ellipse, Polygon etc

- can a telescope be in more than one array, how is its inclusion managed - or is that something for the observatory to do that is no business of Polaris

- ObservingConfiguration has one Instrument, one Telescope and one Backend - what is this config used for, also ObservingMode, and what is Backend?

**Comment by pahjbo** [comment]
Rather than a glossary, I have started a [concepts](https://ivoa.github.io/ProposalDM/concepts/) page within the documentation to describe things in as 'natural language' fashion as possible - currently this has stopped at a fairly high level, but should be extended in future.

As well as this description, there is clearly scope for better descriptions of the concepts within the data model definitions [ProposalDM](https://github.com/ivoa/ProposalDM/blob/master/src/main/vodsl/proposaldm.vodsl) and [ProposalManagementDM](https://github.com/ivoa/ProposalDM/blob/master/src/main/vodsl/proposalManagement.vodsl) - these descriptions are the text that appears in the quotes in the model.

In general the concepts of ObservingConfiguration Instrument Backend and ObservingMode are fairly under specified currently, as they have not really been tested in a variety of circumstances - it needs more work.
